test-cases/load/admission_control_overload_test.yaml

```diff
@@ -0,0 +1,17 @@
+test_duration: 20
+
+prepare_write_cmd: "cassandra-stress user profile=/tmp/cs_profile_background_reads_overload.yaml ops'(insert=10)' no-warmup cl=ONE -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore"
+stress_cmd_r: "cassandra-stress user profile=/tmp/cs_profile_background_reads_overload.yaml ops'(simple1=100)' no-warmup cl=ONE duration=10m -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore"
+stress_cmd_w: "cassandra-stress user profile=/tmp/cs_profile_background_reads_overload.yaml ops'(insert=100)' no-warmup cl=ONE duration=10m -port jmx=6868 -mode cql3 native -rate threads=3000 -errors ignore"
+
+n_db_nodes: 1
+n_loaders: 5
```

Comment:
Increase this to 10 loaders (in order to bypass the scp issue)

Reply:
increased to 12 (might be able to decrease to 10, but need to fine tune using our builders to be sure it will be enough)